dataikuapi/dss/savedmodel.py

```diff
@@ -133,6 +133,12 @@ def import_mlflow_version_from_path(self, version_id, path, code_env_name="INHER
         :param str code_env_name: Name of the code env to use for this model version. The code env must contain at least
                                   mlflow and the package(s) corresponding to the used MLFlow-compatible frameworks.
                                   If value is "INHERIT", the default active code env of the project will be used
+        :param str container_exec_config_name: Name of the containerized execution configuration to use while creating
+                                  this model version. Note that this information is not saved with the model version, so,
+                                  for example, if you evaluate this model version later on, you will have to provide again
```

Comment:
Not sure that I would put this second sentence. Reads a bit alarming to me and the first is clear enough IMHO. WDYT, @fsergot ?